dashboard/app/models/pd/workshop.rb

```diff
@@ -439,6 +439,20 @@ def suppress_reminders?
     ].include? subject
   end
 
+  def self.send_virtual_order_reminder_for_upcoming_in_days(days)
+    # Collect errors, but do not stop batch. Rethrow all errors below.
+    errors = []
+    scheduled_start_in_days(days).each do |workshop|
+      workshop.enrollments.each do |enrollment|
+        email = Pd::WorkshopMailer.teacher_virtual_order_form_reminder(enrollment)
+        email.deliver_now
+      rescue => e
+        errors << "teacher enrollment #{enrollment.id} - #{e.message}"
+      end
+    end
+    raise "Failed to send virtual order form reminders: #{errors.join(', ')}" unless errors.empty?
```

Comment:
For my education, do these errors end up in HoneyBadger?

Reply:
looks like ya, here is an example from a while back: https://codedotorg.slack.com/archives/C55JZ1BPZ/p1527258743000013